DataFormats/Detectors/TOF/src/CalibTimeSlewingParamTOF.cxx

```diff
@@ -50,11 +50,66 @@ CalibTimeSlewingParamTOF::CalibTimeSlewingParamTOF()
   }
 }
 //______________________________________________
```

Comment:
with the huge number of channels you have and a special value -1 it is much faster to do
````
for (int i = 0; i < NSECTORS; i++) {
  memset(mChannelStart[i],-1,sizeof(mChannelStart[i][0]));
}
````

Reply:
I guess you mean:

`memset(mChannelStart[i],-1,sizeof(mChannelStart[i]));`

Reply:
Done

Reply:
actually, I meant ``memset(mChannelStart[i],-1,NCHANNELXSECTOR*sizeof(mChannelStart[i][0]));`` but for the fixed size array also your line will work.

Reply:
Yes, they are equivalent.